### ISSUE TITLE
Search Failing to Complete on One of Two Nodes

### ISSUE THREAD

**Issue by stroomdev00** [gchq/stroom#2635]
b.164
Using standard `Dave Test` search which was completing in seconds in previous versions, still running after 9 mins.  Server Tasks includes the line `Creating extraction tasks - stored data queue size: 0 stream event map size: 0`.

Stack on its way.

**Comment by stroomdev10** [comment]
[stuck_datatest.txt](https://github.com/gchq/stroom/files/7676094/stuck_datatest.txt)

**Comment by stroomdev00** [comment]
Server Tasks associated with the hung search contained the following, all with the same age:

```
Cluster Search - Searching... found 38 docs performed 38 extractions
Search Index shard - Found 38 hits
Search Index Shard - Searching...
Search Index Shard - Searching...
Search Index Shard - Searching...
Search Index Shard - Searching...
Search Index Shard - Searching...
Extraction Task mapper - Creating extraction tasks - stored data queue size: 0 stream event map size: 0
```

**Comment by stroomdev66** [comment]
Fixed by #2637
